eitprocessing/features/breath_detection.py

```diff
@@ -355,3 +355,62 @@ def find_breaths(self, data: np.ndarray) -> list[Breath]:
         ]
 
         return self._remove_breaths_around_invalid_data(breaths, data)
+
+
+    def find_breaths_2d(self, data: np.ndarray) -> np.ndarray:
```

Comment:
It seems this code is mostly a copy of another function. If this function applies an algorithm implemented in an existing function several times, why not call that function several times?